descraibe-frontend/src/App.vue

```diff
@@ -140,6 +140,7 @@
 <script>
 import Chart from 'chart.js/auto'
 import PromptData from '@/data.json'
+import { toHandlers } from 'vue';
```

Comment:
Is this import used anywhere?

Reply:
Removed it in commit 068561f in loss_screen_info